### ISSUE TITLE
Traefix web UI broken with multiple hosts

### ISSUE THREAD

**Issue by Spy-Seth** [containous/traefik#289]
<img width="1194" alt="screenshot 2016-04-07 09 33 36" src="https://cloud.githubusercontent.com/assets/95120/14343860/f57eed70-fca3-11e5-90c0-5171de972780.png">

**Comment by vdemeester** [comment]
Hi @Spy-Seth,

Thanks for the report but could you provide a little bit more info on your setup ? We have an *open* issue on multiple network and such and I guess it could be related to that.

Thanks :wink:

**Comment by Spy-Seth** [comment]
Hi,

I do not put detail because I'm in a simple case. 

To reproduce you just have to create a front end with multiple (long) hostname:

```toml
[frontends]
    [frontends.reader]
    backend = "reader_service"
    passHostHeader = true
        [frontends.reader.routes.main]
        rule = "Host:reader.somedomain.com,reader.test.somedomain.com,reader.staging.somedomain.com,reader.localdev"
```

It will make the corresponding rule cell too big in the UI

**Comment by vdemeester** [comment]
@Spy-Seth thanks :+1: (/me didn't read well the title, I didn't though it was about the UI.. I need anothe coffee :coffee:)

/cc @ldez :angel:

**Comment by ldez** [comment]
Rules with no spaces...
@emilevauge fix in #290
